aiven/client/speller.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021, Aiven, https://aiven.io/
+#
+# This file is under the Apache License, Version 2.0.
+# See the file `LICENSE` for details.
```

Comment:
I'm not a lawyer but I think if this file is under an MIT license, then that should be the only license in the file. It's fine that the code was also modified but it needs to retain its license. I don't think we can dual-license like this, so remove this section.

Reply:
Done!

Reply:
Or should have I keep the first line (Aiven copyright)?